src/components/blocks/block-component-maps/component-map.js

```diff
@@ -82,7 +82,10 @@ export const HtmlTypeComponentMap = Object.freeze({
   [HtmlDataTypes.main]: Main,
   [HtmlDataTypes.article]: Article,
   [HtmlDataTypes.section]: Section,
-  [HtmlDataTypes.aside]: Aside,
+  [HtmlDataTypes.aside]: (props) => {
```

Comment:
This is a bit wonky, but a circular dependency arises when using `Aside` outside of this context - like we do in `MDXWrapper` as a temporary means before we can use cleaner, Markdown-derived `aside` components. The bandage here is to load this on demand instead of webpack's normal module loading.

Alternatively, if the dust settles and GFM asides are still a no go, we can consider React-oriented admonitions like this: https://github.com/nebrelbug/react-admonitions